uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/API/Wrapper/LeaveRequestDates.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+class CRM_HRLeaveAndAbsences_API_Wrapper_LeaveRequestDates implements API_Wrapper {
+
+  /**
+   * Adds Hours to the from_date and to_date fields, in case they're present but
+   * don't have that information.
+   *
+   * For the from_date field, the hour will be set to 00:00:00. For the to_date
+   * field, it will be set to 23:59:59
+   *
+   * @param array $apiRequest
+   *
+   * @return array
+   *   modified $apiRequest
+   */
+  public function fromApiInput($apiRequest) {
+    if($this->canHandleTheRequest($apiRequest)) {
+      $this->prepareDates($apiRequest);
+    }
+
+    return $apiRequest;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function toApiOutput($apiRequest, $result) {
+    return $result;
+  }
+
+  private function canHandleTheRequest($apiRequest) {
+    return $apiRequest['entity'] === 'LeaveRequest' &&
```

Comment:
This condition looks a bit strangely indented, plus it's over 80 characters long, how about splitting it into 

```
    $targetActions = ['get', 'getfull'];
    $isTargetAction = in_array($apiRequest['action'], $targetActions);
    
    return $apiRequest['entity'] === 'LeaveRequest' && $isTargetAction;
```